eng/scripts/get_package_properties.py

```diff
@@ -0,0 +1,12 @@
+import sys
+import glob
+import os
+
+sys.path.append(os.path.join('scripts', 'devops_tasks'))
+from common_tasks import get_package_properties
+
+def get_all_package_properties(service_directory_name):
```

Comment:
While service directory might be the primary filter I think it make sense to accept the entire search_path as a parameter.